### DISCUSSION TITLE
Interacting with another contract

### DISCUSSION THREAD

**Discussion by manaxtech** [Cyfrin/foundry-full-course-cu#615] [Q&A]
Hello, I finished Lesson 4 FundMe smart contract but while I was working on it I deployed a lot of FundMe  smart contract without the withdraw function. Luckily I copied most of the deployed smart contracts addresses, Can someone tell me if and how I can interact with all of deployed smart contract with their address from another contract to withdraw all sepolia ETH out from those contracts to my wallet?

**Comment by theirrationalone** [comment]
# **Impossible**, that's why it's called as Immutable Smart Contracts.
